src/clients/github.js

```diff
@@ -76,37 +105,14 @@ export function requestGitHubToken(username, password, otpCode, note = 'hammer-i
       if (err) {
         reject(filterErrorResponse(err));
       } else {
+        // do something when app is closing
```

Comment:
Can we document what is happening here better or just remove this comment all together.